x/concentrated-liquidity/math/tick.go

```diff
@@ -199,10 +199,9 @@ func CalculatePriceToTick(price osmomath.BigDec) (tickIndex int64, err error) {
 	// N.B. this exists to maintain backwards compatibility with
 	// the old version of the function that operated on decimal with precision of 18.
 	if price.GTE(types.MinSpotPriceBigDec) {
-		// TODO: implement efficient big decimal truncation.
 		// It is acceptable to truncate price as the minimum we support is
 		// 10**-12 which is above the smallest value of sdk.Dec.
-		price = osmomath.BigDecFromDec(price.Dec())
+		price.ChopPrecisionMut(osmomath.PrecisionDec)
```

Comment:
Seems like there are couple of other places in code base where we can use this new method, do we have future plans to change to this method?

Reply:
I am pretty sure yes, it would be out of scope for this issue

Reply:
@pysel next time please make an issue and link it to avoid unaddressed discussions

Reply:
https://github.com/osmosis-labs/osmosis/issues/6467